client/Components/StatBlock.tsx

```diff
@@ -25,6 +25,12 @@ export class StatBlockComponent extends React.Component<
       { name: "Skills", data: statBlock.Skills }
     ];
 
+    const otherScores = [];
```

Comment:
Remove if this is unused

Reply:
I'm kind of hemming and hawwing here. For my purposes, displaying the proficiency bonus and spellcasting ability of PCs in the default stat block is pretty useful, but I didn't know if it would be clutter for the average user. Thoughts?

Reply:
I think it's probably fine to include it, but I noticed that it's currently commented out.